addons/captives/stringtable.xml

```diff
@@ -427,6 +427,10 @@
             <Chinese>投降或無武器狀態</Chinese>
             <Chinesesimp>投降或无武器状态</Chinesesimp>
         </Key>
+        <Key ID="STR_ACE_Captives_KeyComb_Description">
+            <English>Sets the unit under the cursor captive.</English>
+            <German>Nimmt die Einheit vor dem Cursor fest.</German>
+        </Key>¸¸¸¸
```

Comment:
This points got added with the marge, didn't them?

Reply:
Woops, remove them, yeah. 😆